src/sentry/static/sentry/app/views/releases/utils/sessionTerm.tsx

```diff
@@ -0,0 +1,113 @@
+  ),
+  healthy: t('The application exited normally and did not observe any errors'),
+  errored: t('The application exited normally but observed error events while running'),
+  unhandled: t('The application crashed with a hard crashed'),
+};
+
+function getTermDescriptions(platform: PlatformKey) {
+  const technology = platform.includes('javascript') ? platform.split('-')[0] : platform;
+
+  switch (technology) {
+    case 'dotnet':
+    case 'java':
+      return {
+        ...commonTermsDescription,
+        ...mobileTermsDescription,
+      };
+    case 'java-spring':
+    case 'dotnet-aspnetcore':
+      return {
+        ...commonTermsDescription,
+        ...mobileTermsDescription,
+        crashes: t(
+          'A request that resulted in an unhandled exception and hence a Server Error response'
+        ),
+      };
+    case 'android':
+    case 'cordova':
+    case 'react-native':
+    case 'flutter':
+      return {
+        ...commonTermsDescription,
+        ...mobileTermsDescription,
+        crashed: t('An unhandled exception that result in the application crashing'),
+      };
+
+    case 'apple': {
+      return {
+        ...commonTermsDescription,
+        ...mobileTermsDescription,
+        crashed: t('An error that resulted in the application crashing'),
+      };
+    }
+    case 'node':
+    case 'javascript':
+      return {
+        ...commonTermsDescription,
+        crashed: t(
+          "During session an error with mechanism.handled===false occured which is 'onerror' on 'unhandledrejection' handler"
+        ),
+        abnormal: t('Non applicable for Javascript'),
+        healthy: t('No errors captured during session life-time'),
+        errored: t(
+          'During the session at least one error occurred that did not bubble up to the global handlers, not resulting in the application loading process crashing.'
+        ),
+        unhandled:
+          "En error bubbled up to the global 'onerror' or 'onunhandledrejection' handler",
+      };
+    case 'apple-ios':
+    case 'minidump':
+    case 'native':
+      return {
+        ...commonTermsDescription,
+        ...desktopTermDescriptions,
+      };
+    case 'rust':
+      return {
+        ...commonTermsDescription,
+        ...desktopTermDescriptions,
+        crashed: t('The application had an unrecovable error (a panic)'),
+      };
+    default:
+      return {
+        ...commonTermsDescription,
+        crashed: t('Number of users who experienced an unhandled error'),
+        abnormal: t('An unknown session exit'),
+        healthy: mobileTermsDescription.healthy,
+        errored: mobileTermsDescription.errored,
+        unhandled: mobileTermsDescription.unhandled,
+      };
+  }
+}
+
+type Term = keyof ReturnType<typeof getTermDescriptions>;
+
+export function getSessionTermDescription(term: Term, platform: PlatformKey) {
+  return getTermDescriptions(platform)[term];
```

Comment:
Is there a chance that a `term` gets requested that doesn't exist?

Reply:
if the term doesn't exist `undefined` will be returned and that is fine. I will add a couple of tests 😉